manifests/init.pp

```diff
@@ -2,11 +2,58 @@
 #
 # === Parameters:
 #
-# $foreman_url::                URL on which foreman is going to run
+# $admin_username::             Username for the initial admin user
+#
+# $admin_password::             Password of the initial admin user, default is randomly generated
+#
+# $admin_first_name::           First name of the initial admin user
+#
+# $admin_last_name::            Last name of the initial admin user
+#
+# $admin_email::                E-mail address of the initial admin user
+#
+# $db_manage::                  if enabled, will install and configure the database server on this host
+#                               type:boolean
+#
+# $db_type::                    Database 'production' type (valid types: mysql/postgresql/sqlite)
+#
+# $email_source::               template to use for email configuration file
```

Comment:
Move this to advanced, it can't be changed without adding a template to the environment.